docs/coordinates/galactocentric.rst

```diff
@@ -171,7 +171,11 @@ up-to-date list of valid names, see the docstring of
 like ``'pre-v4.0'``, which sets the default parameter values to their original
 definition (i.e. pre-astropy-v4.0) values, and ``'v4.0'``, which sets the
 default parameter values to a more modern set of measurements as updated in
-Astropy version 4.0.
+Astropy version 4.0. Also, custom sets of measurements can be registered to
+`~astropy.coordinates.galactocentric_frame_defaults` and used like the
+built-in options.
+
+`~astropy.coordinates.galactocentric_frame_defaults` also tracks the references for all parameter values, as well as any further specified metadata information.
```

Comment:
```suggestion
`~astropy.coordinates.galactocentric_frame_defaults` also tracks the references (i.e. scientific papers that define the parameter values) for all parameter values, as well as any further specified metadata information.
```
Could you also wrap this line to be consistent with the other text in this file?